tests/test_qpu_devices.py

```diff
@@ -87,7 +87,7 @@ def test_check_connection_provider_no_backend_provided_credentials(self):
         used are correct, check_connection should return True.
         The provider_connected attribute should be updated to True.
         """
-
+        raise ValueError(f"The API_TOKEN: {self.API_TOKEN}")
```

Comment:
What's the purpose of this line? I think it can be located in a better position.